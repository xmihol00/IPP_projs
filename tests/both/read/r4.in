
895
TRue
TRUE
true
