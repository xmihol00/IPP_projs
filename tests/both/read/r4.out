
895
true
true
true
